<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditorToolBar</class>
 <widget class="QWidget" name="EditorToolBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>736</width>
    <height>40</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="StyledAction" name="pushAddSample"/>
   </item>
   <item>
    <widget class="StyledAction" name="pushAddInstrument"/>
   </item>
   <item>
    <widget class="StyledAction" name="pushAddPreset"/>
   </item>
   <item>
    <widget class="StyledAction" name="pushToolBox"/>
   </item>
   <item>
    <widget class="QFrame" name="separator_1">
     <property name="minimumSize">
      <size>
       <width>8</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>8</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="StyledAction" name="pushUndo"/>
   </item>
   <item>
    <widget class="StyledAction" name="pushRedo"/>
   </item>
   <item>
    <widget class="QFrame" name="separator_2">
     <property name="minimumSize">
      <size>
       <width>8</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>8</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="StyledAction" name="pushSave"/>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layoutDisplay"/>
   </item>
   <item>
    <widget class="QFrame" name="separator_3">
     <property name="minimumSize">
      <size>
       <width>8</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>8</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="StyledAction" name="pushShowRecorder">
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="StyledAction" name="pushShowKeyboard">
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>StyledAction</class>
   <extends>QPushButton</extends>
   <header>styledaction.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
